<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntregaUi</class>
 <widget class="QDialog" name="EntregaUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>335</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>331</width>
    <height>335</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>331</width>
    <height>335</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Entrega</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog{
	background-color: rgb(255, 255, 255);
}

QLabel{
font-family: Roboto;
font-style: normal;
font-weight: 450;
font-size: 15px;
letter-spacing: 0.02em;
color: #6B7280;}



/* COMBOBOX */

QComboBox{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
color:#9CA3AF;
border-radius:8px;
padding:10px;
combobox-popup: 0;
border: 2px solid #F3F4F6;

}


QComboBox::drop-down
{
    width: 20px;
    border-left-width: 1px;
	padding: 10px 10px 10px 20px;
 }


QComboBox QAbstractItemView {
	color:#9CA3AF;
	font-size:14px;
	background-color: #F3F4F6;
	selection-background-color:#344647;
	selection-color:#ffffff;
	outline: 0px;
   border: 1px solid #344647;
 border-radius:8px;
padding:10px
}



QComboBox::down-arrow
{
	image: url(:/menu/contraerabajo.png);
     width: 11px;
     height: 11px;
}

QComboBox::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}



QDateEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
color:#9CA3AF;
padding:8px;
border-radius:8px; }

QDateEdit::up-button {
min-width: 25px; min-height: 12px; background-color: #F3F4F6; 
border-radius: 6px;}

QDateEdit::down-button {
min-width: 25px; min-height: 12px; background-color: #F3F4F6; 
border-radius: 6px;}

QDateEdit::up-arrow{
	image: url(:/menu/contraerarriba.png);
     width: 11px;
     height: 11px;
}

QDateEdit::down-arrow{
	image: url(:/menu/contraerabajo.png);
     width: 11px;
     height: 11px;}

QDateEdit::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}


/*  BOTON */

QPushButton{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
color:white;
padding:8px;
background: #3A4F50;
border-radius: 4px;}

QPushButton:hover{background: #344647;
border-radius: 4px; }

QLineEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
border: 2px solid #F3F4F6;
border-radius: 8px;
color:#9CA3AF;
padding:10px}

QLineEdit::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}

</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>4455</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_3{background: #3A4F50 ;border-top-left-radius: 10px;
border-top-right-radius: 10px;}

QLabel{

font-family: Roboto;
font-style: normal;
font-weight: 500;
font-size: 17px;
line-height: 40px;
letter-spacing: 0.02em;
color:#ffffff;

}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="titulo_accion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:19px</string>
    </property>
    <property name="text">
     <string>Nueva Entrega</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="nombre_tutor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>32</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Nueva Afiliaci√≥n</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_closeadmi_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Cerrar</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_closeadmi_3{
font-family: Roboto;
font-weight: normal;
font-size: 14px;
background-color: #d5d5d8;
color: #6B7280;
border-radius:11px
}



#btn_closeadmi_3:hover{background:#ce4d3a ;
border-radius:11px; }


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image/source.qrc">
      <normaloff>:/menu/x.png</normaloff>:/menu/x.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>13</width>
      <height>13</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>54</y>
     <width>331</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{background-color:white}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true">#page{
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <widget class="QDateEdit" name="fecha_informe">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>171</width>
        <height>40</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl9_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Fecha  entrega</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_registrar_informe">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>161</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>567</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Registrar entrega</string>
      </property>
      <property name="icon">
       <iconset resource="../image/source.qrc">
        <normaloff>:/menu/botonadd.png</normaloff>:/menu/botonadd.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>15</width>
        <height>21</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="lbl9_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Mes</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_mes">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>121</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true">#page_2{
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <widget class="QRadioButton" name="rbt_si_ficha">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>SI</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbt_no_ficha">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>NO</string>
      </property>
     </widget>
     <widget class="QLabel" name="pregunta_ficha">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>¬øEntrego ficha?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_registrar_ficha">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>161</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>567</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Registrar entrega</string>
      </property>
      <property name="icon">
       <iconset resource="../image/source.qrc">
        <normaloff>:/menu/botonadd.png</normaloff>:/menu/botonadd.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>15</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true">#page_3{
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <widget class="QRadioButton" name="rbt_no_memorandum">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>NO</string>
      </property>
     </widget>
     <widget class="QLabel" name="pregunta_memorandum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>¬øEntreg√≥ Memorandum?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbt_si_memorandum">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>SI</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_registrar_memorandum">
      <property name="geometry">
       <rect>
        <x>159</x>
        <y>130</y>
        <width>161</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>567</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Registrar entrega</string>
      </property>
      <property name="icon">
       <iconset resource="../image/source.qrc">
        <normaloff>:/menu/botonadd.png</normaloff>:/menu/botonadd.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>15</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl9_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Tipo entrega</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cbo_tipo_entrega">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>311</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maxVisibleItems">
     <number>67</number>
    </property>
    <item>
     <property name="text">
      <string>Informe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ficha</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Memorandum</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../image/source.qrc"/>
 </resources>
 <connections/>
</ui>
